docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Destination type - Email
+nav_order: 1
+parent: monitors
+has_children: false
+---
+
+#  Email as a Destination 
```

Comment:
Sentence case for all headers in the ODFE documentation.

Reply:
Yes I know. Will fix it.